workflow-templates/techdocs.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish TechDocs Site
+
+on:
+  push:
+    branches: [main]
+    # Run only when TechDocs related files are updated.
+    paths:
+      - "docs/**"
+      - "mkdocs.yml"
+
+jobs:
+  publish-techdocs-site:
+    runs-on: ubuntu-latest
+
+    # Use GitHub Organization secrets to set them for all existing and new repositories.
+    env:
+      TECHDOCS_GCS_BUCKET_NAME: ${{ secrets.TECHDOCS_GCS_BUCKET_NAME }}
```

Comment:
I dont see this in Github secrets 🤔 nor the ones below

Reply:
I've created some of them now, still need to get `TECHDOCS_GCS_ACCESS_KEY_ID` from somewhere...

Reply:
This isn't normailly how we handle these things ... The env vars created by core infra are whole JSON objects that contain the private key and a bunch of other stuff, base64-encoded. I can't actually find any references to GCP access key IDs by googling either 🤔 Isn't that an AWS thing?

I found the techdocs CLI a little fiddly to actually consume these credentials - I had to write them to a local path in the script and then set `GOOGLE_APPLICATION_CREDENTIALS=/tmp/creds techdocs-cli publish`. Subshell substitution didn't work for some reason.

Reply:
but basically that file/string contains everything you need to auth, it's like the `.aws/credentials` file, so you don't need more than 1 variable here.